README.md

```diff
@@ -1,63 +1,142 @@
+
+- Accounting
+- Ejector
+
+### Accounting module
+
+Accounting module updates the protocol TVL, distributes node-operator rewards, and processes user withdrawal requests.
+
+**Flow**
+
+The oracle work is delineated by time periods called frames. Oracles finalize a report in each frame.
+The default Accounting Oracle frame length is 225 epochs, which is 24 hours. The frame includes these stages:
+
+- **Waiting** - oracle starts as daemon and wakes up every 12 seconds (by default) in order to find the last finalized slot (ref slot).
+  If ref slot missed, Oracle tries to find previous non-missed slot.
+- **Data collection**: oracles monitor the state of both the execution and consensus layers and collect the data;
+- **Hash consensus**: oracles analyze the data, compile the report and submit its hash to the HashConsensus smart contract;
+- **Core update report**: once the quorum of hashes is reached, meaning more than half of the oracles submitted the same hash,
+  one of the oracles chosen in turn submits the actual report to the AccountingOracle contract, which triggers the core protocol
+  state update, including the token rebase, distribution of node operator rewards, finalization of withdrawal requests, and
+  deciding whether to go in the bunker mode.
+- **Extra data report**: an additional report carrying additional information. This part is required.
+  All rewards to modules are distributed in this Oracle’s phase. Can be submitted in chunks.
+
+### Ejector module
 
-Oracle daemon for Lido decentralized staking service. Collects and reports CL states to the Lido dApp contracts running on Ethereum EL side.
+Ejector module ejects Lido validators when the protocol requires additional funds to process user withdrawals.
 
-There are two types of oracles:
-  - **accounting:**
-      reports info about validators count, balances (validators, withdrawal vault, EL rewards vault), count of exited validators
-  - **ejector:**
-      reports info about next validators to be ejected (to initiate exit from CL)
+**Flow**
+
+- Finds out how much ETH is needed to cover withdrawals.
+- Predicts how much ETH will be rewarded in each time period.
+- Figures out when the next validator will be removed.
+- Changes lido validators into simpler code and share the update.
 
 # Usage
 
+## Requirements
+
+The oracle daemon requires a node with Execution and Consensus layers. We highly recommend using
+[geth](https://geth.ethereum.org/docs/install-and-build/installing-geth#run-inside-docker-container),
+[Lighthouse](https://lighthouse-book.sigmaprime.io/docker.html#using-the-docker-image) and
+[Erigon](https://github.com/ledgerwatch/erigon#getting-started).
+
+### Machine requirements
+
+- vCPUs - 8
+- Memory - 16 GB
+
+### Dependencies
+
+#### Execution Client Node
+
+To prepare the report, Oracle fetches up to 10 days old events. To perform this, Oracle needs [full](https://ethereum.org/en/developers/docs/nodes-and-clients/#full-node) execution node.
```

Comment:
The oracle also makes historical requests for balance data or makes simulated reports on historical blocks. This requires an Archive node. I think it's better not to write anywhere that Full node is needed, as this could lead to a misinterpretation of the requirements.

I would suggest writing that we need 2 weeks of archived data